packages/destination-actions/src/destinations/fullstory/index.ts

```diff
@@ -0,0 +1,56 @@
+import type { DestinationDefinition } from '@segment/actions-core'
+import { defaultValues } from '@segment/actions-core'
+import type { Settings } from './generated-types'
+import identifyUser from './identifyUser'
+import { listOperationsRequestParams, deleteUserRequestParams } from './request-params'
+import { dataRegions } from './data-regions'
+
+const destination: DestinationDefinition<Settings> = {
```

Comment:
This was generated by the Segment CLI, after which I customized these values borrowing heavily from the FullStory browser destination and adjusting where it seemed sensible (specifying cloud mode, adding settings values for API key and region, etc.).